File: Admin/Admin.Server/Dto/Document/DocumentViewData.cs
using PlatformShared.Dto;

namespace Admin.Server.Dto.Document
{
    /// <summary>
    /// Document view data
    /// 文档浏览数据
    /// </summary>
    public record DocumentViewData : SystemDocumentViewData
    {
        /// <summary>
        /// Organization id
        /// 机构编号
        /// </summary>
        public int? OrgId { get; init; }
    }
}

File: Admin/admin.client/src/api/dto/document/DocumentViewData.ts
import { SystemDocumentViewData } from "@etsoo/smarterp-core";

/**
 * Document view data
 * 文档浏览数据
 */
export type DocumentViewData = SystemDocumentViewData & {
  /**
   * Organization id
   * 机构编号
   */
  orgId?: number;
};
